doc/nrf/libraries/zigbee/index.rst

```diff
@@ -9,7 +9,74 @@ See the :ref:`ug_zigbee` user guide for an overview of the technology.
 Zigbee libraries are a set of components that can be directly reused in Zigbee applications.
 They implement the default behavior of a Zigbee device and are used in :ref:`zigbee_samples` in this SDK.
 
-For information on how to use the supplied libraries, see :ref:`ug_zigbee_configuring_libraries`.
+The following table lists libraries enabled by default for each available Zigbee sample (✔) or available through an overlay file (``**``):
+
+.. list-table::
+    :widths: auto
+    :header-rows: 1
+
+    * - Library name
+      - :ref:`Light bulb <zigbee_light_bulb_sample>`
+      - :ref:`Light switch <zigbee_light_switch_sample>`
+      - :ref:`Network coordinator <zigbee_network_coordinator_sample>`
+      - :ref:`NCP <zigbee_ncp_sample>`
+      - :ref:`Shell (sample) <zigbee_shell_sample>`
+      - :ref:`Template <zigbee_template_sample>`
+    * - ZBOSS OSIF
+      - ✔
+      - ✔
+      - ✔
+      - ✔
+      - ✔
+      - ✔
+    * - Shell (library)
+      -
+      -
+      -
+      -
+      - ✔
+      -
+    * - Application utilities (default signal handler)
+      - ✔
+      - ✔
+      - ✔
+      -
+      - ✔
+      - ✔
+    * - Error handler
+      - ✔
+      - ✔
+      - ✔
+      -
+      - ✔
+      - ✔
+    * - FOTA
+      -
+      - ``**``
```

Comment:
Looks like dot in final docs instead of `*`

Reply:
Fixed